src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/cas1/Cas1PremisesDayTransformer.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.cas1
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas1OutOfServiceBedSummary
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas1PremiseCapacity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas1PremisesDaySummary
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas1SpaceBookingDaySummary
+import java.time.LocalDate
+
+@Component
+class Cas1PremisesDayTransformer {
+
+  fun toCas1PremisesDaySummary(
+    date: LocalDate,
+    premisesCapacity: Cas1PremiseCapacity,
+    spaceBookings: List<Cas1SpaceBookingDaySummary>,
+    outOfServiceBeds: List<Cas1OutOfServiceBedSummary>,
+  ) =
+    Cas1PremisesDaySummary(
+      forDate = date,
+      previousDate = date.minusDays(1),
+      nextDate = date.plusDays(1),
+      capacity = premisesCapacity.capacity.first(),
```

Comment:
i think it may be cleaner to just pass the single capacity entry in as the caller knows that one and only one day was requested